### DISCUSSION TITLE
EMS-ESP32 Ver. 3.0.0. - how to send MQTT commands

### DISCUSSION THREAD

**Discussion by SpaceTeddy** [emsesp/EMS-ESP32#1048] [General]
Hi guy's,

I updated to Ver. 3.0.0 and try to send mqtt commands to the gateway. Basically I saw, that there is a discussing about mqtt structure, but I tried two ways to enable the boiler with one time WW charging:
a) ems-esp/boiler/wwonetime 1
b) ems-esp/boiler {"cmd": "wwonetime" ,"data":"on", "hc":1}

may you provide me some examples to set boiler wwonetime and thermostat day/night temperatur for hc1?

thx,
Chris

**Comment by proddy** [comment]
I just tested with the latest dev and those commands work (wwonetime).

**Comment by MichaelDvP** [comment]
`wwonetime`was always a bit tricky. There are a lot of discussions how it works depending on boiler, boiler settings, temperature-levels, thermostat, etc. The syntax is ok, the needless hc-settings is ignored, data in both topics can be `"on"`, `1`, `"1"`, `"true"`, or `true`.

**Comment by SpaceTeddy** [comment]
Hi all,
works with latest dev v3.0.1b3. 

you can close the issue
